src/lib/connectors/idm.js

```diff
@@ -145,9 +145,14 @@ usersClient.disableUser = partialRight(setEnabled, false);
  */
 usersClient.enableUser = partialRight(setEnabled, true);
 
+const getKPIAccessRequests = async => {
```

Comment:
Is this one deprecated by `src/lib/connectors/idm/kpi-reporting.js`

Reply:
removed